lib/jwt_sessions/store_adapters/abstract_store_adapter.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+require 'redis'
+
+module JWTSessions
+  module StoreAdapters
+    class AbstractStoreAdapter
+      class << self
+        def instance_by_symbol(adapter, options = {})
+          camelized_adapter = adapter.to_s.split('_').map(&:capitalize).join
+          adapter_class_name = "JWTSessions::StoreAdapters::#{camelized_adapter}StoreAdapter"
+          Object.const_get(adapter_class_name).instance(options)
+        end
+
+        def instance(token_prefix: nil, **store_options)
```

Comment:
We can use default arg value instead of `... || ...` below

```suggestion
        def instance(token_prefix: JWTSessions.token_prefix, **store_options)
```